MPF.Frontend/Tools/ProtectionTool.cs

```diff
@@ -304,7 +304,82 @@ public static string SanitizeFoundProtections(IEnumerable<string> foundProtectio
             // SafeDisc
             if (foundProtections.Any(p => p.StartsWith("SafeDisc")))
             {
-                if (foundProtections.Any(p => Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled) && !p.StartsWith("Macrovision Protection File")))
+                // Confirmed this set of checks works with Redump entries 11347, 13230, 28257, 31824, 52606, 57721, 58455, 58573, 62935, 63941, 64255, 65569, 66005, 70504, 73502, 74520, 79729, 83468, 98589, and 101261.
+
+                // Best case scenario for new SafeDisc 2+: A full SafeDisc version is found in a line starting with "Macrovision Protect Application". All other SafeDisc detections can be safely scrubbed.
+                // TODO: Scrub "Macrovision Protected Application, " from before the SafeDisc version. Also scrub the "SafeDisc SRV Tool APP" portion of the detection when not needed.
+                if (foundProtections.Any(p => Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled) && p.StartsWith("Macrovision Protected Application")))
+                {
+                    foundProtections = foundProtections.Where(p => !p.StartsWith("Macrovision Protection File"))
+                        .Where(p => !p.StartsWith("Macrovision Security Driver"))
+                        .Where(p => p != "SafeDisc")
+                        .Where(p => !p.StartsWith("Macrovision Protected Application, Macrovision Protected Application [Version Expunged]"))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}-[0-9]\.[0-9]{2}\.[0-9]{3}", RegexOptions.Compiled)))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}\+", RegexOptions.Compiled)))
+                        .Where(p => !(Regex.IsMatch(p, @"SafeDisc [0-9]\.[0-9]{2}\.[0-9]{3}\/4\+", RegexOptions.Compiled)))
```

Comment:
Even though `/4+` is currently the most common, if you're already using regex, it may be safer to change this to `\/[0-9]\+`

Reply:
This filter is also needed after each of the existing `\+` statements. Its own `if` statement is probably warranted as well.

Reply:
> Even though `/4+` is currently the most common, if you're already using regex, it may be safer to change this to `\/[0-9]\+`

There are currently no other numbers that get reported as`/X+` for SafeDisc. I could add it if you insist, but at the moment it wouldn't make any difference except for making the code very slightly less easy to understand, IMO

Reply:
> This filter is also needed after each of the existing `\+` statements. Its own `if` statement is probably warranted as well.

Can you clarify exactly what filter you mean is needed?